clmm/utils.py

```diff
@@ -782,8 +782,7 @@ def integrand(z_i, z_cl=z_cl, cosmo=cosmo):
     B_mean = quad(integrand, zmin, zmax)[0] / quad(z_distrib_func, zmin, zmax)[0]
     return B_mean
 
-def compute_beta_s_mean(z_cl, z_inf, cosmo, zmax=10.0, delta_z_cut=0.1, zmin=None,
-                        z_distrib_func=None):
+def compute_beta_s_mean(z_cl, z_inf, cosmo, zmax=10.0, delta_z_cut=0.1, zmin=None, z_distrib_func=None, weights_option=False, z_src = None, shape_weights = None):
```

Comment:
I think you can remove the `weights_option` argument and just apply weight when `shape_weights` is provided

Reply:
Done